caluma/workflow/filters.py

```diff
@@ -53,6 +56,48 @@ class CaseFilterSet(MetaFilterSet):
 
     has_answer = HasAnswerFilter(document_id="document__pk")
     status = case_status_filter(lookup_expr="in")
+    order_by_question_answer_value = CharFilter(
+        method="filter_order_by_question_answer_value",
+        label=(
+            "Expects a question slug. If the slug is prefixed with a dash, "
```

Comment:
Nitpick: I think `dash` refers to another character ("longer minus", in german "Gedankenstricht / Halbgeviertstrich") - what we're expcing is called "hyphen" or "minus sign".
https://en.wikipedia.org/wiki/Dash
https://en.wikipedia.org/wiki/Hyphen

Reply:
Absolutely!